docs/Websocket_endpoints.md

```diff
@@ -159,7 +159,7 @@ visual   | Object | Room visuals (contents currently unknown #FIXME)
 
 ```javascript
 // Subscription
-api.socket.subscribe('room:W7N7', event => console.log(JSON.stringify(event.data)));
+api.socket.subscribe('room:shard0/W7N7', event => console.log(JSON.stringify(event.data)));
```

Comment:
Not sure if `Note that adding a shard to a server not expecting it may cause an error.` is entirely true...